airbyte-config/config-models/src/main/resources/types/WebhookOperationConfigs.yaml

```diff
@@ -0,0 +1,28 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/WebhookOperationConfigs.yaml
+title: WebhookOperationConfigs
+description: List of configurations for webhook operations
+additionalProperties: false
+# NOTE: we have an extra layer of object nesting because the generator has some weird behavior with arrays.
```

Comment:
curious - can you clarify this extra layer?

Reply:
Linked a related github issue here.